.pre-commit-config.yaml

```diff
@@ -31,6 +31,6 @@ repos:
           - --confidence-level=medium
           - --skip=B608
   - repo: https://github.com/syslabcom/slc.pre-commit
-    rev: main
+    rev: v1.0.0
```

Comment:
```suggestion
    rev: 1.0.0
```

Reply:
FWIW github claims that “It’s common practice to prefix your version names with the letter v.”